tests/ui/target-feature/feature-hierarchy.aarch64-sve2.stderr

```diff
@@ -1,7 +0,0 @@
-warning: target feature `neon` cannot be disabled with `-Ctarget-feature`: this feature is required by the target ABI
```

Comment:
This warning disappears because `sve2` implies `neon`, so `-neon,+sve2` actually doesn't have adverse ABI effects.